Lib/test/test_gdb.py

```diff
@@ -54,6 +54,16 @@ def get_gdb_version():
 
 PYTHONHASHSEED = '123'
 
+
+def cet_cf_protection():
+    cflags = sysconfig.get_config_var('CFLAGS')
+    flags = cflags.split()
+    return (('-mcet' in flags)
+            and any(flag.startswith('-fcf-protection') for flag in flags))
+
+CET_CF_PROTECTION = cet_cf_protection()
```

Comment:
Nitpick:
CET - Control-flow enforcement technology
CF - Control-flow enforcement technology
CET_CF_PROTECTION - Control-flow enforcement technology control-flow protection

Pick just one.

Reply:
>  CET_CF_PROTECTION - Control-flow enforcement technology control-flow protection

Oops :-) I renamed the constant.